### DISCUSSION TITLE
I have setup the complete setup but my Grunt file is giving error.

### DISCUSSION THREAD

**Discussion by syed-umer-syer** [opensourcepos/opensourcepos#3889] [Q&A]
I have setup the complete setup but my Grunt file is giving error and i am not able to complete the grunt build. I have added the project in root directory (htdocs).
any help would be greatly appreciated, I have attached error screenshots as well as the text for error.
**ERROR** 
`Running "wiredep:task" (wiredep) task

Running "bower_concat:all" (bower_concat) task
>> Can’t detect any .tmp/opensourcepos_bower.js on main files for "bootswatch" component. You should explicitly define it via bower_concat’s mainFiles option. See Readme for details.
File tmp/opensourcepos_bower.js created.
>> Can’t detect any .tmp/opensourcepos_bower.css on main files for "bootswatch" component. You should explicitly define it via bower_concat’s mainFiles option. See Readme for details.
File tmp/opensourcepos_bower.css created.

Running "bowercopy:targetdistjqueryui" (bowercopy) task
Warning: public/bower_components/jquery-ui/themes/base/jquery-ui.min.css was not found Use --force to continue.

Aborted due to warnings.`
**SCREEN SHOTS**
![Screenshot from 2023-12-09 02-22-50](https://github.com/opensourcepos/opensourcepos/assets/62714295/d58e974c-db2d-41b6-b81f-96fa631a4605)
![Screenshot from 2023-12-09 02-22-36](https://github.com/opensourcepos/opensourcepos/assets/62714295/c2556c5b-ac29-4613-bba4-306c473458f9)
![Screenshot from 2023-12-09 02-22-03](https://github.com/opensourcepos/opensourcepos/assets/62714295/cd69fd21-8ea1-49d2-9b0f-c1a8a3a905d9)

**Comment by jekkos** [comment]
Hi, from the logs it seems that the jquery ui css file is missing for some reason. Have you ran a bower install ? It needs to install the dependencies to be able to copy the css first.

**Reply by syed-umer-syer** [reply]
Hi Thank you for your response, I apologies for not showing complete information, but, yes, I had installed them. but for reference for this post I have installed node_modules and bower components again but it still does not resolve the previous issue mentioned above.
**screen shots of node modules and bower components installation**
![Screenshot from 2023-12-09 14-01-56](https://github.com/opensourcepos/opensourcepos/assets/62714295/767a434b-22a4-443c-a695-bf1aebc115bc)
![Screenshot from 2023-12-09 14-01-44](https://github.com/opensourcepos/opensourcepos/assets/62714295/bc3d2666-2102-4a12-8ad5-a5189a38b290)
![Screenshot from 2023-12-09 14-01-24](https://github.com/opensourcepos/opensourcepos/assets/62714295/f30bd193-aa99-41df-9b26-994bf92e384e)
![Screenshot from 2023-12-09 14-01-09](https://github.com/opensourcepos/opensourcepos/assets/62714295/78d8d261-8910-49e2-a117-032a689dd363)
![Screenshot from 2023-12-09 14-00-49](https://github.com/opensourcepos/opensourcepos/assets/62714295/676b9d53-6aef-40ff-89a4-f864761b09bc)
![Screenshot from 2023-12-09 14-00-17](https://github.com/opensourcepos/opensourcepos/assets/62714295/1368b62f-f285-4c72-9948-3b1fb6d0dab1)
![Screenshot from 2023-12-09 13-59-08](https://github.com/opensourcepos/opensourcepos/assets/62714295/72d938c0-9368-48a4-9199-d2445af15cda)
**screen shots of bower components folder**
![Screenshot from 2023-12-09 14-03-00](https://github.com/opensourcepos/opensourcepos/assets/62714295/621d5d2d-3028-4894-aa64-be3037ac60cf)
![Screenshot from 2023-12-09 14-02-49](https://github.com/opensourcepos/opensourcepos/assets/62714295/b68f145c-ec12-456f-8c85-ddfbba341248)
